app/javascript/packs/application.js

```diff
@@ -2,4 +2,28 @@ require("@rails/ujs").start()
 require("turbolinks").start()
```

Comment:
even though `;` are nos really necessary in js, the style convention is to use them anyway (this could change). If you were using `;`s in this project let's maintain a standard

Reply:
Okey, ive corrected it:  3649a33